mirage/scenarios/collections.ts

```diff
@@ -94,37 +70,135 @@ export function collectionModerationScenario(server: Server, currentUser: ModelI
+            license: licensesAcceptable[0],
+            title: `Pending Project Request - ${suffix}`,
+            collectionSubmissionActionArgument: buildInitialCollectionSubmissionActionArguments(
+                CollectionSubmissionActionTrigger.Submit,
+            ),
+        }  as ProjectBuilderArgument, CollectionSubmissionReviewStates.Pending);
+    });
 
-    acceptedProject({
-        server,
-        currentUser,
-        collection: primaryCollection,
-        license: licensesAcceptable[0],
-        title: 'Accepted Project',
-    } as ProjectModel);
+    /**
+     * Accepted
+     */
+    [1,2].forEach((suffix: number) => {
+        projectBuilder({
+            server,
+            currentUser,
+            collection: primaryCollection,
+            license: licensesAcceptable[0],
+            title: `Accepted Project - ${suffix}`,
+            collectionSubmissionActionArgument: buildInitialCollectionSubmissionActionArguments(
+                CollectionSubmissionActionTrigger.Accept,
+                'You really make this project great',
+            ),
+        }  as ProjectBuilderArgument, CollectionSubmissionReviewStates.Accepted);
+    });
 
-    removedProject({
-        server,
-        currentUser,
-        collection: primaryCollection,
-        license: licensesAcceptable[0],
-        title: 'Removed Project',
-    } as ProjectModel);
+    /**
+     * Removed By Project Admin
+     */
+    [1,2,3,4].forEach((suffix: number) => {
+        projectBuilder({
+            server,
+            currentUser,
+            collection: primaryCollection,
+            license: licensesAcceptable[0],
+            title: `Removed Project - ${suffix}`,
+            collectionSubmissionActionArgument: buildInitialCollectionSubmissionActionArguments(
+                CollectionSubmissionActionTrigger.AdminRemove,
+                'Thanks for being part of our project',
+            ),
+        }  as ProjectBuilderArgument, CollectionSubmissionReviewStates.Removed);
+    });
 
-    rejectedProject({
-        server,
-        currentUser,
-        collection: primaryCollection,
-        license: licensesAcceptable[0],
-        title: 'Rejected Project',
-    } as ProjectModel);
+    /**
+     * Removed By Collection Moderator
+     */
+    [1,2,3].forEach((suffix: number) => {
+        projectBuilder({
+            server,
+            currentUser,
+            collection: primaryCollection,
+            license: licensesAcceptable[0],
+            title: `Removed Project - ${suffix}`,
+            collectionSubmissionActionArgument: buildInitialCollectionSubmissionActionArguments(
+                CollectionSubmissionActionTrigger.ModeratorRemove,
+                'Thanks for being part of our project',
+            ),
+        }  as ProjectBuilderArgument, CollectionSubmissionReviewStates.Removed);
+    });
+
+    /**
+     * Rejected
+     */
+    [1,2,3,4,5,6,7,8,9,10].forEach((suffix: number) => {
```

Comment:
My only caution here (and with the others in this file) is that if you find that the app is starting to take an excessive amount of time to start up, it's probably all of these extra items being created. You're not creating a lot of related resources to go with (like contributors or files) so it's probably fine, but if the app does take forever to start up when using these scenarios, you might want to slim these down.